README.md

```diff
@@ -21,7 +21,7 @@ terraform {
   required_providers {
     uptimerobot = {
       source = "vexxhost/uptimerobot"
-      version = "0.7.0"
+      version = "0.8.0"
```

Comment:
Considering Terraform Registry [generates docs at each release](https://www.terraform.io/registry/providers/docs#publishing), this will likely need to change to something like `0.8.1` along with a release either way.